test/mpi.py

```diff
@@ -195,7 +195,7 @@ def test_select_device(self):
     def test_make_node_file(self):
         """test if the nodefile is made and if it contains the same amount of nodes as mpinodes"""
         mpi_instance = getinstance(mpim.MPI, Local, MympirunOption())
-        mpi_instance.make_node_file()
+        mpi_instance.make_machine_file()
```

Comment:
should we rename `make_node_file` to `make_mpdboot_file` then?

In any case, rename the test too here (& fix the docstring), since it's quite confusing now :)